Cargo.toml

```diff
@@ -85,9 +85,9 @@ members = [
 	"frame/elections",
 	"frame/election-provider-multi-phase",
 	"frame/election-provider-support",
-	"frame/example",
-	"frame/example-offchain-worker",
-	"frame/example-parallel",
+	"frame/examples/basic",
+	"frame/examples/offchain-worker",
+	"frame/examples/parallel",
```

Comment:
Can we add some information that this is highly experimental and not really supported? Maybe in the crate level docs?

Reply:
I updated with a note that these example pallets are not meant to be used in production in [this commit](https://github.com/paritytech/substrate/pull/10215/commits/38c52309a805c9b369b20d3b08750df2bf81094a).